components/expenses/CreateExpenseForm.js

```diff
@@ -134,6 +139,12 @@ class CreateExpenseForm extends React.Component {
       });
       return false;
     }
+    if (expense.type === '' && !expense.type) {
+      this.setState({
+        error: intl.formatMessage(this.messages['error.expenseTypeMissing']),
```

Comment:
Good job translating the error message, there are too many places where we forget to do so :+1: